universal-adapter-kotlin/src/main/java/com/appunite/universal_adapter_kotlin/BaseViewHolderManager.kt

```diff
@@ -0,0 +1,40 @@
+package com.appunite.universal_adapter_kotlin
+
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.annotation.LayoutRes
+import com.jacekmarchwicki.universaladapter.BaseAdapterItem
+import kotlin.reflect.KClass
+
+open class BaseViewHolderManager<T : BaseAdapterItem>(
+        @LayoutRes private val layoutRes: Int,
```

Comment:
I wonder if this a good approach to always requiring `layoutRes`. What if someone doesn't want to use an XML file, and wants just creating the view by code?

Reply:
I've never had to create an item view another way, have you? Anyway, I added an abstract method to create the view so it is possible now to take another approach to that.